### DISCUSSION TITLE
PWA disabling

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#17841] [Feature Requests]
### Summary

Is it possible to disable the progressive web app support?

### Motivation

When creating a link on iOS on the home it creates an pwa. Then I can’t see the link.

### Detailed Design

.

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

.

### Alternatives

.

### Adoption Strategy

.

**Comment by connorwinston** [comment]
Directus does not have PWA support. I believe you are referring to `Add to Home Screen` which is just a native iOS feature. This is not related to Directus. https://support.apple.com/guide/iphone/bookmark-favorite-webpages-iph42ab2f3a7/ios
